test/config_parser_SUITE.erl

```diff
@@ -932,7 +945,284 @@ s2s_max_retry_delay(_Config) ->
                   #{<<"s2s">> => #{<<"max_retry_delay">> => 120}}),
     err_host_config(#{<<"s2s">> => #{<<"max_retry_delay">> => 0}}).
 
-%% Helpers for 'listen' tests
+modopts(Mod, Opts) ->
+    [#local_config{key = {modules, ?HOST}, value = [{Mod, Opts}]}].
+
+%% modules
+
+%% ---------------------------------------------------------------------------
```

Comment:
I am not a fan of this kind of ascii art, for me it only clutters the code. What is more, this formatting is inconsistent with the rest of this file. I'd avoid these decorations if possible. I'd like to hear a second opinion here though.

Reply:
I second that, the consistency being the most important reason